jumpscale/clients/explorer/models.py

```diff
@@ -412,6 +471,19 @@ class Container(Base):
     capacity = fields.Object(ContainerCapacity)
     info = fields.Object(ReservationInfo)
 
+    def resource_units(self):
+        cap = self.capacity
+        resource_units = ResourceUnitAmount()
+        resource_units.cru = cap.cpu
+        resource_units.mru = round(cap.memory / 1024)
```

Comment:
mru is a float so rounding here only makes sense if we want to limit the amount of significant digits like it is done on the next line imo